scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -814,16 +814,32 @@ let notUsingGitPush1by1 =
 
         let json = GitHubApiCall url
 
-        json.Contains "\"check_suites\":[]"
+        not(json.Contains "\"check_suites\":[]")
     )
-    |> Seq.contains true
 
-if notUsingGitPush1by1 then
+let notActivatedGitHubActions = not(Seq.contains true hasCiStatus)
+
+if notActivatedGitHubActions then
+    let errMsg =
+        sprintf
+            """
+Please activate GitHub Actions in your repository. Click on the 
+"Actions" tab at the top of the repository page and click on the 
+"I understand my workflows, go ahead and enable them" button. 
+"""
+
+    Console.Error.WriteLine errMsg
+    Environment.Exit 1
+
+let notUsedGitPush1byt =
+    Seq.contains true hasCiStatus && Seq.contains false hasCiStatus
```

Comment:
@realmarv and after the changes above, you can use the variable `gitHubActionsEnabled` here instead of violating DRY